spec/factories/users.rb

```diff
@@ -0,0 +1,18 @@
+FactoryGirl.define do
+
+  factory :user do
+    email { Faker::Internet.email }
+    username { Faker::Name.first_name }
+    role 'user'
+    password 'qqqqqq'
+    password_confirmation 'qqqqqq'
+  end
+
+  factory :admin, class: User do
```

Comment:
for admin please use `factory :admin, parent: :user { admin: true }`

Reply:
i didn't see this  message, id did -   trait :admin  do   role :admin  end
factory :admin, parent: :user { admin: true } - also will it work? Because user has column - role, and I have enum in my model: enum role: [:user, :admin]